Sim/SimG4Components/src/SimG4GeantinoDeposits.cpp

```diff
@@ -3,13 +3,85 @@
 // Geant4
 #include "G4VModularPhysicsList.hh"
 #include "G4Geantino.hh"
+#include "G4ChargedGeantino.hh"
 #include "G4VUserPhysicsList.hh"
 #include "G4VProcess.hh"
 #include "G4VContinuousProcess.hh"
 #include "G4ProcessManager.hh"
 #include "G4ProcessType.hh"
 
 
+ 
```

Comment:
I actually forgot this exists. However, I think the name ` SimG4GeantinoDeposits` is quite misleading, what do you think? Wouldn't `SimG4GeantinoPhysicsList` be better?

Reply:
Funny, I would say it's just the other way around. `PhysicsList` suggests that it includes some physics processes, but in the end it just lets geantinos deposit energy in sensitive volumes, so I thought the name is fairly descriptive. I can change it though, if you are not convinced!

Reply:
The way I am looking is that G4 requires a user to give a physics list (of processes that may happen, no matter if it is "true" physics or "fake" physics). Therefore we either give e.g. FTFP_BERT or we give that Geantinos. I am not really sure what is the best name, I just wanted to make it more clear that is is the file user must put to "physicslist" property of SimG4Svc... We actually do have yet another list we use for testing and it is called `SimG4TestPhysicsList` and it kills particles at their origin, so kind of non-physical as well...

Reply:
I see your point, agreed!